code/modules/mapping/mapping_helpers.dm

```diff
@@ -236,25 +236,39 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 	late = TRUE
 	icon_state = "deadbodyplacer"
 	var/bodycount = 2 //number of bodies to spawn
+	var/no_random = FALSE // set it TRUE in mapping if you want to put it into a specific container (when you don't want a random morgue)
```

Comment:
This is not changed anywhere and currently a constant
Also put the comment above the var follow by 3 / (instead of 2)

Reply:
I am not sure what `This is not changed anywhere and currently a constant` means

Reply:
By that I mean this isn't ever set to true. Or at least I couldn't find an instance of it being true.

Reply:
I still don't get what you want

Reply:
okay I'll put it simpler.
It's not used anywhere or set to a different value on any child objects.